includes/definitions/chatsworth.yaml

```diff
@@ -0,0 +1,8 @@
+os: chatsworth
+text: 'Chatsworth PDU'
+type: power
+icon: chatsworth
+nobulk: 1
+discovery:
+    - sysObjectId:
+        - .0.10.43.6.1.4.1
```

Comment:
Unfortunately, yes.. I will try to contact them about it but I'd be surprised if they actually do anything about it